eng/common/templates/steps/test-images-windows-client.yml

```
@@ -1,9 +1,12 @@
 steps:
 - powershell: |
     if ("${{ eq(variables['System.TeamProject'], 'public') }}" -eq "False") {
-      $optionalTestArgs="-Registry $env:ACR_SERVER -RepoPrefix $env:STAGINGREPOPREFIX -TestCategory $env:TESTCATEGORY"
+      $optionalTestArgs="-Registry $env:ACR_SERVER -RepoPrefix $env:STAGINGREPOPREFIX"
     } else {
-      $optionalTestArgs="-IsLocalRun -TestCategory $env:TESTCATEGORY"
+      $optionalTestArgs="-IsLocalRun"
+    }
+    if ($repoTestArgs) {
+      $optionalTestArgs=$optionalTestArgs + "-TestCategory $env:REPOTESTARGS"
```

I would expect this to be: 

```suggestion
      $optionalTestArgs += " $env:REPOTESTARGS"
```

The important difference is that the other repos that are utilizing the **_common_** infrastructure should be able to use the $env:REPOTESTARGS to specify a completely different argument to the test script.

---

Understood.